pkg/operator/operator.go

```diff
@@ -343,7 +339,7 @@ func (o *Operator) handleSingleNode4Dot11Upgrade() error {
 
 // ensureDefaultIngressController creates the default ingresscontroller if it
 // doesn't already exist.
-func (o *Operator) ensureDefaultIngressController(infraConfig *configv1.Infrastructure) error {
+func (o *Operator) ensureDefaultIngressController(infraConfig *configv1.Infrastructure, ingressConfig *configv1.Ingress) error {
```

Comment:
Are these changes still needed if `setDefaultPublishingStrategy` handles defaulting when `spec.endpointPublishingStrategy.loadBalancer.providerParameters.aws.type` is omitted?

Reply:
anyway changes done